idalib/src/idb.rs

```diff
@@ -379,6 +381,29 @@ impl IDB {
         }
     }
 
+    pub fn strings(&self) -> StringList {
+        StringList::new(self)
+    }
+
+    pub fn get_string(&self, index: usize) -> Option<String> {
+        let addr = self.strings().get_item_addr(index)?;
+        let len = self.strings().get_item_length(index);
+
+        let bytes = self.get_bytes(addr, len);
+
+        Some(String::from_utf8_lossy(&bytes).escape_debug().to_string())
+    }
+
+    pub fn ea2str(&self, ea: Address) -> Option<String> {
```

Comment:
I'd move this to `StringList` as `get_by_address`

Reply:
This function actually just converts an address into its rich string representation. For instance:
```
ea2str(0x9141) = ".rodata:0000000000009141"
```
Its name in the IDA API is somewhat misleading and that's why I implemented it in the first place. Then I noticed it doesn't do what I thought it did and it doesn't have much to do with strings in the `StringList`, so I moved it to a `conversions` module that we can populate in the future with other similar utility functions. We could also remove it to keep idalib cleaner if you prefer.

Reply:
Hmm maybe we can call it something less misleading on our side—I’ve got no issue keeping it in that case :)

Reply:
Done. I'll implement other conversion utility functions that make sense when I have time (e.g., some of those from <https://cpp.docs.hex-rays.com/group__conv.html>).